lib/index.coffee

```diff
@@ -104,4 +104,5 @@ module.exports = (opts) ->
     get_output_paths = (files, prefix) ->
       @util.files(files).map (f) =>
         filePath = @util.output_path(f.relative).relative
-        path.join(prefix, filePath.replace(path.extname(filePath), '.js'))
+        fN = path.join(prefix, filePath.replace(path.extname(filePath), '.js'))
```

Comment:
Not if you build it on Windows. It will give you something like *\js\base.js* in your HTML. HTML should always have forward slashes, isn't it?